src/citrine/informatics/scores.py

```diff
@@ -23,12 +23,12 @@ class Score(PolymorphicSerializable['Score']):
     def get_type(cls, data):
         """Return the subtype."""
         return {
-            'MLI': MLIScore,
-            'MEI': MEIScore
+            'MLI': LIScore,
```

Comment:
Note that this makes the change non-breaking wrt the API.

Reply:
I've added a comment to that affect in the PR description, because I think that's the most important thing for @andyczerwonka . This user-facing change does not compel the backend code or the API to change.